src/main/java/uk/gov/ons/ctp/response/collection/exercise/lib/common/redis/SessionResolver.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.ons.ctp.response.collection.exercise.lib.common.redis;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SessionResolver {
```

Comment:
The naming of this feels wrong to me.  This isn't resolving a session, it's just a key deleter (in its current state).   How is this class going to be used in the future?  Do you have one of these classes per thing you want to do? (i.e., one for deleting, another for adding, etc) or per group of thing? (i.e., one for setting/removing keys from exercise endpoint calls, another for caching survey or instrument data, etc)